parsely/src/test/java/com/parsely/parselyandroid/EventsBuilderTest.kt

```diff
@@ -0,0 +1,216 @@
+package com.parsely.parselyandroid
+
+import android.content.Context
+import android.provider.Settings
+import androidx.test.core.app.ApplicationProvider
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.MapAssert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+internal class EventsBuilderTest {
+
```

Comment:
Minor (🔍): Consider removing this extra line here.

PS: I usually don't case, but by now I have used to the fact that our static analysis tooling is always complaining on that, thus I am now complaining about that too... 😛

Reply:
You motivate me to introduce detekt! Thanks, lines removed :)

Reply:
Detekt! 😄 💪 🤣 

FYI: Actually, this warning that we keep getting on all repos (expect `WCAndroid`) is `CheckStyle` related: `Checkstyle: Empty line not allowed after brace` 🤷 🤷 🤷